actions/websocket_actions.jsx

```diff
@@ -193,16 +198,15 @@ export function unregisterPluginReconnectHandler(pluginId) {
 }
 
 function restart() {
-    reconnect(false);
+    reconnect();
 
     // We fetch the client config again on the server restart.
     dispatch(getClientConfig());
 }
 
-export function reconnect(includeWebSocket = true) {
```

Comment:
I forgot to mention it earlier, but we never actually call `reconnect` without passing `false` for `includeWebSocket`, so I just removed that unused code path